src/components/DiffingResult/index.jsx

```diff
@@ -95,16 +95,28 @@ function DiffingResult() {
 
     const newCanvas = initCanvas();
     newCanvas.on("mouse:wheel", ev => {
```

Comment:
- `P2`
선언부와 비즈니스로직은 한칸 개행해주시면 더 좋을 것 같아요!

```suggestion
    const newCanvas = initCanvas();
    
    newCanvas.on("mouse:wheel", ev => {
```